### ISSUE TITLE
plugins.d not included

### ISSUE THREAD

**Issue by badfiles** [ansible-collections/community.zabbix#875]
Default plugins config directory (6.0, 6.2) not included in agent role for linux (master branch)

From default config:
```
# Include configuration files for plugins
Include=./zabbix_agent2.d/plugins.d/*.conf
```
For windows I believe same issue exists.

**Comment by pyrodie18** [comment]
Related to #960

**Comment by solick** [comment]
is there a solution or workaround for this?

**Comment by pyrodie18** [comment]
Closed out as off #1335
